### DISCUSSION TITLE
errors in webui and image-to-image

### DISCUSSION THREAD

**Discussion by nekoyarouu** [hlky/stable-diffusion#282] [Q&A]
Hi everyone! 
This is what I get in the end when I start webui.cmd
![1 1](https://user-images.githubusercontent.com/112880328/188472220-7ee8c806-0aa3-44dd-a55d-e9886fda43b1.jpg)
And when I try to generate an image in image-to-image mode I get this error 
![2 2](https://user-images.githubusercontent.com/112880328/188472253-666eefbe-3892-412c-a19c-c33c27504371.jpg)
Text-to-image works fine.

**Comment by SafentisFox** [comment]
Are you perhaps using the unstable dev version of webui found here https://github.com/hlky/stable-diffusion-webui instead of the stable release found here https://github.com/hlky/stable-diffusion?

**Reply by nekoyarouu** [reply]
> Are you perhaps using the unstable dev version of webui found here https://github.com/hlky/stable-diffusion-webui instead of the stable release found here https://github.com/hlky/stable-diffusion?

No, I've used the link from the installation guide.

**Comment by ReinerBforartists** [comment]
I can confirm the problem with https://github.com/hlky/stable-diffusion . I ran yesterday into it after fetch and pull.

The workaround is to switch to mask mode, paint a bit (it has no effect anyways, and seems to be broken too), switch back, and now imgtoimg should work again.

I have seen this report now more than once now. Across both repos. I would highly suggest to use just one repo, and work in development branches. And maybe somebody could sort out the double bug reports. Also, imho bug reports does not belong into discussions. But a proper task in the issues tracker.

Kind regards
Reiner
